cloud/scope/cluster.go

```diff
@@ -197,6 +201,11 @@ func (s *ClusterScope) IsVnetManaged() bool {
 	return s.Vnet().ID == "" || s.Vnet().Tags.HasOwned(s.ClusterName())
 }
 
+// IsIPV6Enabled Returns if IpV6 is enabled
+func (s *ClusterScope) IsIPV6Enabled() bool {
```

Comment:
nit: there is a mix of `Ipv6`, `IPV6` and `IPv6` in the variable names and function signatures. I think we should probably stick to just using one - `IPv6`?